### PR TITLE
[tronador] Update tronador docs 

### PR DESCRIPTION
**Pull request by MuneebAijaz**

### CODE REVIEW

**Comment by stakater-user** [comment]
@MuneebAijaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-307-288a3add`

**Comment by stakater-user** [comment]
@MuneebAijaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-307-488ac0d7`

File: content/sre/tronador/tronador_config.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        username: username
+      type: Opaque
+    - kind: Secret
+      apiVersion: v1
+      metadata:
+        name: common-secret-2
+      data:
+        password: password
+        username: username
+      type: Opaque
```
Comment (ShahidHussain-srti) [inline content/sre/tronador/tronador_config.md]: use a different resource for this second example

**Review by ShahidHussain-srti — APPROVED** [review]

**Comment by stakater-user** [comment]
@MuneebAijaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-307-e4a73f37`

**Comment by stakater-user** [comment]
@MuneebAijaz Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.238`

**Pull request merged**